src/compiler/core.ts

```diff
@@ -2749,6 +2749,5 @@ export function isNodeLikeSystem(): boolean {
     // and definitely exist.
     return typeof process !== "undefined"
         && !!process.nextTick
-        && !(process as any).browser
-        && typeof module === "object";

```

Comment:
In the new version, `module` is a parameter and is always provided. This check would no longer do anything. I'm not sure that it really matters that we don't have this.

Reply:
Actually, I'm going to switch this back to the same definition it used to be, e.g. checking for `require`. The comment I left above about `require` stopped being correct at some point as I refined our output bundle format to rewrite `require` temporarily.

Reply:
(I don't believe this will end up mattering in practice, but that's why we merge these sorts of things early in the cycle...)